backend/src/libs/types/repository.type.ts

```diff
@@ -3,7 +3,7 @@ type Repository<T = unknown> = {
   findAll(): Promise<T[]>;
   create(payload: unknown): Promise<T>;
   update(payload: unknown): Promise<T>;
-  delete(): Promise<boolean>;
+  delete(id: number): Promise<number>;
```

Comment:
```suggestion
  delete(id: number): Promise<boolean>;
```

Reply:
@Liza-Veis @v0ldemar01 said me to return boolean from service in this [comment](https://github.com/BinaryStudioAcademy/bsa-2023-calmpal/pull/307#discussion_r1328080490)

Reply:
ok, let's leave it